plutus-playground/plutus-playground-client/bower.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "plutus-playground-client",
+  "description": "",
+  "main": "",
+  "authors": [
+    "Kris Jenkins <kris.jenkins@tweag.io>"
+  ],
+  "license": "AllRightsReserved",
```

Comment:
Is this the same BSD3 license as the rest of the codebase?

Reply:
fixed